packages/fxa-payments-server/server/lib/server.js

```diff
@@ -155,7 +182,7 @@ module.exports = () => {
             return proxyResData;
           }
           const body = proxyResData.toString('utf8');
-          return injectHtmlConfig(body, CLIENT_CONFIG, {});
+          return injectHtmlConfig(body, CLIENT_CONFIG, {}, flowMetricsData());
```

Comment:
Also, have you considered just calling the content server's [GET /metrics-flow](https://github.com/mozilla/fxa/blob/1cb0d0a4e8104b700a78a414fcc2ecf96e0fd6fc/packages/fxa-content-server/server/lib/routes/get-metrics-flow.js#L83) endpoint like other services do? We could add a new [form_type](https://github.com/mozilla/fxa/blob/1cb0d0a4e8104b700a78a414fcc2ecf96e0fd6fc/packages/fxa-content-server/server/lib/routes/get-metrics-flow.js#L66) and doing so would allow us to avoid the [proliferation of the HMAC key](https://github.com/mozilla/fxa/pull/2801/files#diff-36e2db7c1336ac4eec3760f309b6f90fR118).